aesop/spectra.py

```diff
@@ -79,6 +79,47 @@ def __init__(self, wavelength=None, flux=None, name=None, mask=None,
         self.time = time
         self.continuum_normalized = continuum_normalized
 
+    def flux_calibrate(self, fluxed_spec, polynomial_order, plots=False):
+        """
+        Interpolate high-res spectrum to low-res flux calibrated spectrum, then fit
+        the ratio with a polynomial to flux calibrate. Returns polynomial coefficients
+
+        Parameters
+        ----------
+        fluxed_spec : `~aesop.Spectrum1D`
```

Comment:
Nitpicky: can we change this var name to `flux_calibrated_spectrum`? Explicit > implicit.